update analytics doc for amplify-js v6 (#6153)
aws-amplify/docs: src/fragments/lib/analytics/js/autotrack/page-tracking.mdx

    // OPTIONAL, provide the URL for the event.
    urlProvider:  () => {
      // the default function
      return window.location.origin + window.location.pathname;
    }
  }
});
```

You can turn it off by:
```javascript
import { configureAutoTrack } from 'aws-amplify/analytics';

configureAutoTrack({
  enable: false,
  type: 'pageView'
});
```

## Page Event Tracking

If you want to track user interactions with elements on the page, you can use this feature. All you need to do is attach the specified selectors to your dom element and turn on the auto tracking.

To turn it on:

```javascript
import { configureAutoTrack } from 'aws-amplify/analytics';

configureAutoTrack({
  // REQUIRED, turn on/off the auto tracking
  enable: true,
  // REQUIRED, the event type, it's one of 'event', 'pageView' or 'session'
  type: 'event',
  // OPTIONAL, the customizable options for the event type.
  options: {
    // OPTIONAL, the attributes of the event
    attributes: {
      customizableField: 'attr'
    },
    // OPTIONAL, events you want to track, by default is 'click'
    events: ['click'],

    // OPTIONAL, the prefix of the selectors, by default is 'data-amplify-analytics-'

>     // in order to avoid collision with the user agent, according to https://www.w3schools.com/tags/att_global_data.asp
>     // always put 'data' as the first prefix

Reviewer: Nit:
```suggestion
    // Per https://www.w3schools.com/tags/att_global_data.asp, always start the
    // the prefix with 'data' to avoid collisions with the user agent
```

    selectorPrefix: 'data-amplify-analytics-'
  }
});
```

For example:

```html
<!-- you want to track this button and send an event when it is clicked -->
<button
  data-amplify-analytics-on="click"
  data-amplify-analytics-name="click"
  data-amplify-analytics-attrs="attr1:attr1_value,attr2:attr2_value"
/>
```

When the button above is clicked, an event will be sent automatically. This is equivalent to doing:

```html
<script>
  import { record } from 'aws-amplify/analytics';
  var sendEvent = function() {
    record({
      event: {
        name: 'click',
        attributes: {
          attr: 'attr', // the default ones
          attr1: attr1_value, // defined in the button component
          attr2: attr2_value // defined in the button component
        }
      }
    });
  };
</script>
<button onclick="sendEvent()" />
```

> You can turn it off by:

Reviewer: ```suggestion
This behavior can be disabled by calling `configureAutoTrack`:
```
